src/relay/op/contrib/ethosu/depthwise.cc

```diff
@@ -132,6 +136,18 @@ bool EthosuDepthwiseConv2DRel(const Array<Type>& types, int num_inputs, const At
   const auto* param = attrs.as<EthosuDepthwiseConv2DAttrs>();
   ICHECK(param != nullptr) << "EthosuDepthwiseConv2DAttrs cannot be nullptr.";
 
+  DataType ofm_dtype;
+
+  if (param->ofm_dtype == "int8") {
```

Comment:
It might be clearer if we use a dictionary here

Reply:
Ack